sync/gh.py

```diff
@@ -117,6 +117,14 @@ def close_pull(self, pr_id):
         # issue.add_to_labels("mozilla:backed-out")
         pr.edit(state="closed")
 
+    def is_approved(self, pr_id):
+        pr = self.get_pull(pr_id)
+        reviews = pr.reviews()
+        for review in reviews:
+            if review.state == "APPROVED":
```

Comment:
The reviews are in chronological order so I think you want the last review to be approved, not the first.